Tests/TuistGeneratorTests/Generator/BuildPhaseGeneratorTests.swift

```diff
@@ -86,6 +86,39 @@ final class BuildPhaseGeneratorTests: XCTestCase {
         }
     }
 
+    func test_generateSourcesBuildPhase_whenLocalizedFile() throws {
+        // Given
+        let target = PBXNativeTarget(name: "Test")
+        let pbxproj = PBXProj()
+        pbxproj.add(object: target)
+
+        let files: [AbsolutePath] = [
+           "/path/sources/Base.lproj/OTTSiriExtension.intentdefinition",
+           "/path/resources/en.lproj/OTTSiriExtension.intentdefinition",
```

Comment:
To be honest I didn't run the tests at all, because I was receiving "No such module 'TuistSupportTesting'" error. Now I have manually added this dependency to some target and fixed the tests.